### PR TITLE
Bug 1725329 - Fixed Compare View Permalink functionality

### PR DESCRIPTION
**Pull request by sorintoma**

### CODE REVIEW

File: ui/App.jsx [hunk 1]
```diff
@@ -100,7 +109,7 @@ const withFavicon = (element, route) => {
 };
 
 const App = () => {
-  updateOldUrls();
+  checkIfOldUrl();
```
Comment (airimovici) [inline ui/App.jsx:112]: I'm not a fan on the name of this, I would use `updateUrls`

**Review by airimovici — APPROVED** [review]

File: ui/App.jsx [hunk 2]
```diff
@@ -55,15 +56,24 @@ const updateOldUrls = () => {
     if (index >= 2 && updates.pathname !== subRoute && subRoute !== '/jobs') {
       updates.pathname += subRoute;
     }
-  } else if (search.length) {
+  }
+
+  if (search.length) {
```
Comment (airimovici) [inline ui/App.jsx:61]: I don't think this change is intended. We should keep this behavior under the `else`, as before

**Comment by airimovici** [comment]
It would help if we had a unit test that tests the permalink urls not to be modified anymore

**Review by beatrice-acasandrei — APPROVED** [review]

**Comment by airimovici** [comment]
Hi @sarah-clements, can you take a look at these changes? This fixes the permalink functionality from compare view

**Review by sarah-clements — APPROVED** [review]

**Pull request merged**